### DISCUSSION TITLE
Icon request: icon-cvv (cv2 / cvc / cvv2 / cvc2)

### DISCUSSION THREAD

**Discussion by razor-rich** [FortAwesome/Font-Awesome#21173] [Icon Requests]
We do a lot of ecommerce sites and an icon similar to this is something i have to create time again. Any thoughts on including Cvv and Cvv2 icons

![image](https://cloud.githubusercontent.com/assets/5901199/13113028/44f7d04c-d585-11e5-941f-340f82e59fb8.png)

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
